File: 1010/1010.py
cod1 , qtd1 , valor1 = input().split()
cod2 , qtd2 , valor2 = input().split()

cod1 = int(cod1)
cod2 = int(cod2)
qtd1 = int(qtd1)
qtd2 = int(qtd2)
valor1 = float(valor1)
valor2 = float(valor2)

tot1 = valor1 * qtd1
tot2 = valor2 * qtd2

print("VALOR A PAGAR: R$ %.2f" % (tot1 + tot2))

File: URI/1010/1010.c
#include<stdio.h>

int main(){

    double valor1 , valor2 , p1 , p2 ;
    int codigo1 , codigo2;
    int qtd1 , qtd2 ;

    scanf("%d %d %lf",&codigo1 ,&qtd1 , &valor1);
    scanf("%d %d %lf", &codigo2, &qtd2, &valor2);

    p1 = qtd1 * valor1;
    p2 = qtd2 * valor2;
    p1 = p1 + p2;
    
    printf("VALOR A PAGAR: R$ %.2lf\n", p1);
    return 0;
}